### PR TITLE
Added the Ui

### PR DESCRIPTION
**Pull request by thankgodr**
This PR includes

1. Calculator UI 
2. Reuseable components
3. Code Liting

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (aimalamiri) [inline src/App.js]: Good job! but it's required for this task to use only class components, and you'll have the opportunity to implement the functional components in the future, so please make sure to update this.

File: src/App.test.js [hunk 2]
```diff
@@ -3,6 +3,6 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+  const linkElement = screen.getByText(/9/i);
   expect(linkElement).toBeInTheDocument();
```
Comment (aimalamiri) [inline src/App.test.js]: This file is part of the CRA, so kindly make sure to remove it as well.

File: src/components/ResultView.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ResultView extends React.Component {
+  render() {
+    return (
+            <div className="col-md-12 resultview">
+              <p className='result'>{this.props.result}</p>
+            </div>
```
Comment (aimalamiri) [inline src/components/ResultView.js:9]: It looks fantastic, but to better match the design I'd highly recommend increasing the height of your result view and making it as much as your button's height.

image


[Optional] You'll need to write and read from this component, so don't you think it must be an `input` tag with the type of `text or number`?

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @thankgodr, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights

- The application layout is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right Git Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,10 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```
Comment (omarsalem7) [inline src/App.js]: - [x] As stated before by the previous reviewer, kindly convert your App component from a functional to a class-based component.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @thankgodr, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

Almost near

 ## Highlights
✔️ All linter checks pass
✔️ Descriptive pull request

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] I highly recommended using an updated readme template from Microverse for your project to make it professional.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (danmainah) [reply src/App.js]: @thankgodr great job implementing the changes requested however you forgot to convert your app component to be class-based, kindly look into it

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @thankgodr , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [optional] Kindly ensure your readme file uses the correct template, here is the recommended template[microverse template](https://github.com/microverseinc/readme-template)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @thankgodr   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**